src/gui/updownratiodlg.cpp

```diff
@@ -43,34 +46,78 @@ UpDownRatioDlg::UpDownRatioDlg(bool useDefault, qreal initialValue,
     if (useDefault) {
         ui->useDefaultButton->setChecked(true);
     }
-    else if (initialValue == -1) {
+    else if (initialRatioValue == -1 && initialTimeValue == -1) {
```

Comment:
Heh you're right. It's for readability. I've seen it suggested so many times i thought it is for sure in the guidelines. Either way i've seen at least one more and gonna point it out in a while. Feel free to ignore them if noone else says something about it.